pkgs/tools/security/doas/default.nix

```diff
@@ -23,6 +24,7 @@ stdenv.mkDerivation rec {
 
   configureFlags = [
     (lib.optionalString withTimestamp "--with-timestamp") # to allow the "persist" setting
+    (lib.optionalString (!withPAM) "--without-pam")
     "--pamdir=${placeholder "out"}/etc/pam.d"
```

Comment:
Should we also gate this with `lib.optionalString withPAM "..."`?

Reply:
Well, upstream build system defaults to "--with-pam", so it is not strictly necessary.